development-updates.md

```diff
@@ -20,7 +20,11 @@ Phase one is the very beginning of the cohort. The first few weeks are dedicated
 | [Ashely Yan](https://github.com/AshliaYan)                   |                                                                 | [Update 1](https://hackmd.io/@Ashelyyan/SyRBmmorC)              | [Update 2](https://hackmd.io/@Ashelyyan/BJPffq-IR)              |
 | [Ashen](https://github.com/y1cunhui)                         | [Update 0](https://hackmd.io/@gr3y/SkjEawHBC)                   |                                                                 |                                                                 |
 | [Bastin](https://github.com/Inspector-Butters)               | [Update 0](https://hackmd.io/@Bastin/HJ6hOLQHC)                 | [Update 1](https://hackmd.io/@Bastin/HyM3AmnrA)                 | [Update 2](https://hackmd.io/@Bastin/H1JgDZLU0)                 |
+<<<<<<< HEAD
 | [BobLiu](https://github.com/Akagi201)                        | [Update 0](https://hackmd.io/@Akagi201/epf-cohort5-week0)       | [Update 1](https://hackmd.io/@Akagi201/epf-cohort5-week1)       | [Update 2](https://hackmd.io/@Akagi201/epf-cohort5-week2)       |
+=======
+| [BobLiu](https://github.com/Akagi201)                        | [Update 0](https://hackmd.io/@Akagi201/epf-cohort5-week0)       | [Update 1](https://hackmd.io/@Akagi201/epf-cohort5-week1)       |                                                                 |
+>>>>>>> origin/project-proposal
```

Comment:
The PR contains many of these artifacts left from some conflict. Please remove all of them, make sure the rest of document just follows the main